CHANGES.rst

```diff
@@ -8,6 +8,20 @@ https://rosettasciio.readthedocs.io/en/latest/changes.html
 
 .. towncrier release notes start
 
+API Removal
+-----------
```

Comment:
As this is relevant to hyperpsy users and the API was in hyperspy before, would it be better to move it to the corresponding entry of hyperpsy 2.0 changelog?

Reply:
Good point, we might want to just duplicate it there, in case migrating HyperSpy users end up looking here?

Reply:
see https://github.com/hyperspy/hyperspy/pull/3002

Reply:
I still find odd to have a `API removal` section for a first release! I would remove this section and a normal changelog entry - the description on its own is good.

Reply:
I moved it to a regular towncrier entry for the PR and checked the formatting locally.